apps/fileworker/src/routes/front/upload/+page.svelte

```diff
@@ -110,13 +122,15 @@
 					d="M7 16a4 4 0 01-.88-7.903A5 5 0 1115.9 6L16 6a5 5 0 011 9.9M15 13l-3-3m0 0l-3 3m3-3v12"
 				/>
 			</svg>
-			{#if !candidateFile}
+			{#if candidateFiles.length === 0}
 				<p class="mb-2 text-sm text-gray-500 dark:text-gray-400">
 					<span class="font-semibold">Click to upload</span>
 					 or drag and drop.
 				</p>
 			{:else}
-				<p class="mb-2">{candidateFile.name}</p>
+				{#each candidateFiles as candidateFile}
```

Comment:
I wonder if we need to use Keyed Each block:
https://svelte.dev/docs/svelte/each#keyed-each-blocks
Not sure honestly, let me know!

Reply:
Maybe? but idk it seems to work